app/src/main/java/io/intrepid/contest/screens/conteststatus/ContestStatusActivity.java

```diff
@@ -19,14 +19,21 @@
 public class ContestStatusActivity extends BaseMvpActivity<ContestStatusContract.Presenter>
         implements ContestStatusContract.View, ContestStatusActivityContract {
 
+    private static final String JUDGE_JUST_SUBMITTED_KEY = "judge_submitted_key";
+
+    public static Intent makeIntent(Context context, boolean judgeSubmitted) {
+        return new Intent(context, ContestStatusActivity.class).putExtra(JUDGE_JUST_SUBMITTED_KEY, judgeSubmitted);
+    }
+
     public static Intent makeIntent(Context context) {
-        return new Intent(context, ContestStatusActivity.class);
+        return new Intent(context, ContestStatusActivity.class).putExtra(JUDGE_JUST_SUBMITTED_KEY, false);
```

Comment:
we can DRY the code up a bit by just calling the other make intent: return makeIntent(context, false)